packages/ringcentral-widgets/containers/WelcomePage/index.js

```diff
@@ -46,6 +50,14 @@ function mapToFunctions(_, {
     onLoginButtonClick() {
       oAuth.openOAuthPage();
     },
+    onSignUpButtonClick() {
+      const url = 'https://www.ringcentral.com/office/plansandpricing.html';
+      if (chrome && chrome.tabs) {
```

Comment:
Please don't add platform related codes in common widgets library

Reply:
done

Reply:
done